README.md

```diff
@@ -235,7 +255,7 @@ on: [push]
 jobs:
   run:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml:0-dvc2-base1
+    container: docker://ghcr.io/iterative/cml:0-dvc2-base1
```

Comment:
If this accessible outside GH? If not I would set aside as an GH optimisation.

Reply:
> _If this accessible outside GH?_

Of course!

Reply:
its faster than docker hub? or only faster if used in GH?

Reply:
I haven't performed any benchmarks, but I would assume both. If you're curious, feel free to add a card to the backlog, open an issue or whatever and I'll run some tests one of these days.

Reply:
The major drawback that I see also is that is not good with GL or others drivers. He had issues about this kind of issues in the past

Reply:
> _The major drawback that I see also is that is not good with GL or others drivers. He had issues about this kind of issues in the past. I prefer it to be agnostic and an improvement for GH_

What do you mean?

Reply:
❓

Reply:
I prefer to always use resources in any example that are not a plus over a driver. When asking GH or GL to showcase and support CML  they did not like to see the other around

Reply:
So we are giving all of them the same love and care without any advantage

Reply:
Well, ghcr.io happens to be offered by GitHub, but it's just an awesome public container registry, like Docker Hub. 🤷🏼‍♂️  If you prefer to recommend Docker Hub as the primary registry, that's also fine.

Reply:
Yes, because of BB and GL, despite that we can still make the GH examples using it while in BB and GL we MUST user dockerhub

Reply:
I deliberately only put `ghcr.io/iterative` in the GH-specific example. Notice that elsewhere where it could be ambiguous what driver the end user may be interested in using, `dvcorg` is mentioned first.